test/src/test/java/io/seata/saga/engine/db/StateMachineDBTests.java

```diff
@@ -107,6 +107,26 @@ public void testSimpleStateMachineWithChoice() {
         System.out.println("====== cost :" + cost);
     }
 
+    @Test
+    public void testSimpleStateMachineWithChoiceNoDefault() {
+
+        long start = System.currentTimeMillis();
+
+        Map<String, Object> paramMap = new HashMap<>();
+        paramMap.put("a", 3);
+
+        String stateMachineName = "simpleChoiceNoDefaultTestStateMachine";
+
+        try {
+            stateMachineEngine.start(stateMachineName, null, paramMap);
+        } catch (EngineExecutionException e) {
+            e.printStackTrace(System.out);
+            Assertions.assertNotNull(e);
```

Comment:
Useless code?

Reply:
No, this unit test is expected to throw an exception

Reply:
I mean this line of code.I mean this line of the code：`Assertions.assertNotNull(e);`

Reply:
fixed